core/src/apps/cosmos/transaction.py

```
@@ -224,13 +226,18 @@
             raise wire.DataError("invalid payload")
-
-        return Transaction(MessageArgs(**j))
+        return Transaction(
+            MessageArgs(
+                account_number=j["account_number"],
+                chain_id=j["chain_id"],
+                fee=j["fee"],
+                memo=j["memo"],
+                msgs=j["msgs"],
+                sequence=j["sequence"],
+            )
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)